stories/molecules/TimePicker/TimePicker.stories.jsx

```diff
@@ -37,7 +37,7 @@ export default {
         minuteFormat: ['mm', 'm'],
```

Comment:
Please left only one format as you done for hourFormat and secondFormat as well.